packages/beacon-node/src/db/repositories/index.ts

```diff
@@ -4,7 +4,7 @@ export {BlobSidecarsArchiveRepository} from "./blobSidecarsArchive.js";
 export {BlockRepository} from "./block.js";
 export {BlockArchiveRepository} from "./blockArchive.js";
 export type {BlockArchiveBatchPutBinaryItem, BlockFilterOptions} from "./blockArchive.js";
-export {StateArchiveRepository} from "./stateArchive.js";
```

Comment:
also remove `stateArchive.ts` because we don't use it anymore? if yes I suppose we need a new cli to prune it, this helps users a lot of saving disc space